src/main/java/edu/ksu/canvas/model/assignment/AssignmentGroup.java

```diff
@@ -0,0 +1,57 @@
+package edu.ksu.canvas.model.assignment;
+
+import edu.ksu.canvas.annotation.CanvasField;
+import edu.ksu.canvas.model.BaseCanvasModel;
+
+public class AssignmentGroup extends BaseCanvasModel{
+
+    Long id;
+    String name;
+    Long position;
+    Long group_weight;
```

Comment:
We use camelCase naming conventions, not underscores. Gson translates it to underscores in any JSON we get from or send to Canvas. Same goes for method names.